eisbuk-admin/src/pages/attendance_printable/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { useSelector } from "react-redux";
+
+import Container from "@material-ui/core/Container";
+
+import AppbarAdmin from "@/components/layout/AppbarAdmin";
+import DateNavigation from "@/components/atoms/DateNavigation";
+import AttendanceSheet from "@/components/atoms/AttendanceSheet/AttendanceSheet";
+import AttendanceSheetSlot from "@/components/atoms/AttendanceSheet/AttendanceSheetSlot";
```

Comment:
Ok...so this is where the `index.ts` file (in each component folder) is of use. What you wan't to do is: 
- Instead of importing `@/components/AttendanceSheet/AttendanceSheet` you want to import `@/components/AttendanceSheet` (index is automatic entry point of the folder, and the `default export` of `(...)/AttendanceSheet/index.ts` is the `AttendanceSheet` component.
- Also, instead of importing from `"@/components/atoms/AttendanceSheet/AttendanceSheetSlot"` you want to do `import {AttendanceSheetSlot} from "@/components/atoms/AttendanceSheet"` - since there can only be one default export, you simply do a named export in `(...)/AttendanceSheet/index.ts` file: 
```typescript
// ...
import AttendanceSheetSlot from "./AttendanceSheetSlot"
export { AttendnaceSheetSlot }
``` 
**As some rules of thumb:** 
1. Each file exported from the component folder should be exported from `index.ts` file, while the others are (sort of) private to that folder, this gives us a better overview of what is used locally and what is imported from elsewhere
2. In accordance to `1.`, you should only import from the folder (i.e.`@/components/AttendanceSheet`), and under the hood from `index.ts` file located in that folder, not dig through the folders files but rather, if you need some of the files within a folder, add the said file to `index.ts` file's exports.